roles/glance/tasks/main.yml

```diff
@@ -30,6 +30,22 @@
   file: dest=/var/cache/glance state=directory mode=0700 owner=glance
         group=glance
 
+- block:
+  - name: change the selinux label for the glance image dirs
+    command: semanage fcontext -a -t glance_var_run_t "{{ glance.state_path }}(/.*)?"
```

Comment:
we need to check output of these commands so we can accurately show changed|ok|failed status for the tasks.